openstack/keystone/templates/etc/_policy.yaml.tpl

```diff
@@ -8,13 +8,13 @@
 "service_or_admin": "rule:admin_required or rule:service_role"
 
 #
-#"owner": "user_id:%(user_id)s"
```

Comment:
Can you please a comment with the default policy so we can compare our overwritten rules with the original ones (even if they are the same)?

Reply:
I can, but it will be different from other entries (see "service_role" or "admin_required" above on line 5). Do you still think i need to do it?

Reply:
Indeed, I think it should be added for those lines, too, then :)